include/enums/c_tensor_type.h

```diff
@@ -33,15 +33,15 @@
 #define SMARTREDIS_CTENSORTYPE_H
 
 typedef enum {
-	c_undefined = 0,
-    c_dbl       = 1,
-    c_flt       = 2,
-    c_int8      = 3,
-    c_int16     = 4,
-    c_int32     = 5,
-    c_int64     = 6,
-    c_uint8     = 7,
-    c_uint16    = 8
+	c_invalid_tensor = 0,
+    c_dbl     = 1,
```

Comment:
Align ``c_cbl`` and other enums with ``c_invalid_tensor``.  I think it is just this group misaligned.  Previous c enum declarations look good.

Reply:
fixed -- a tab got inserted instead of spaces